rustls-libssl/Makefile

```diff
@@ -76,6 +76,19 @@ target/VERSION: ALWAYS
 	echo "This is rustls-libssl `git describe --always`\nDate: `date`\nIncorporating:" > $@
 	$(CARGO) tree >> $@
 
+test-package: test-package-deb
+
+test-package-deb: package-deb
+	mkdir -p target/dist/test/ubuntu-22
```

Comment:
won't flag them individually, but consider `--parents`, and `--tag` instead of `-p` and `-t`.